doc/nrf/includes/filter.js

```diff
@@ -0,0 +1,86 @@
+<script>
+
+$(document).ready(function(){
+
+  /** Function that either shows all versions or hides all except one **/
+  function displayVersion(theVersion) {
+
+    switch(theVersion) {
+      case "all":
+        $("#known-issues dl").show();
+        break;
+      default:
+        $("#known-issues dl").hide();
+        $("#known-issues dl."+theVersion).show();
+    };
+
+   $("[name='versions']").val(theVersion);
+
+  };
+
+  /** Display classes of the format vXXX as tag vX.X.X **/
+  $("#known-issues dl").each(function() {
+
+    var thisDl = $(this);
+
+    $(this).attr("class").split(" ").forEach(function(className) {
+
+       var URL = window.location.href.split('?')[0];
+
+       if(RegExp('v\\d{3}').test(className)) {
```

Comment:
I think this is OK for now, but if feels very fragile; a better alternative would have been to have used `_` to separate the parts of a version, eg, `v1_3_1`.

Reply:
That's a lot nicer - fixed (but with "-", since Sphinx converts underscores in classes, for whatever reason ...).